recipe/meta.yaml

```diff
@@ -7,9 +7,8 @@ package:
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  git_url: https://github.com/hmacdope/mdanalysis.git
```

Comment:
You need to point to package here I think, otherwise it doesn't know where to find the toml.